android/src/app/organicmaps/MwmApplication.java

```diff
@@ -164,7 +176,6 @@ private void initNativePlatform() throws IOException
     Config.setStoragePath(writablePath);
     Config.setStatisticsEnabled(SharedPropertiesUtils.isStatisticsEnabled(this));
 
-    Editor.init(this);
```

Comment:
Where is this call now?

Reply:
Nowhere. This call was needed only for calling `OsmUploadWork.startActionUploadOsmChanges(Context)` from `AppBackgroundTracker.onTransit(boolean)`. `OsmUploadWork.startActionUploadOsmChanges(Context)` is now called directly from `MwmApplication.onBackground()`. 

https://github.com/organicmaps/organicmaps/blob/d89c0b45b91e7b3766388de0d6b89acfd6438974/android/src/app/organicmaps/editor/Editor.java#L51-L54